vite.config.ts

```diff
@@ -4,9 +4,6 @@ import { svelte } from "@sveltejs/vite-plugin-svelte";
 import wasmPack from "vite-plugin-wasm-pack";
 
 export default defineConfig({
-  test: {
```

Comment:
The type checker let us know that nothing actually pays attention to this! It's a `vitest` property, but we're not using that anymore; just raw playwright